src/modules/ROOT/pages/changelog/202303/02.json

```diff
@@ -40,7 +41,7 @@
   "changed": [
     {
       "what": [
-        "the way reminders in {APP_NAME_COMPOSE} look and behave. The new interface provides an easier way for the user to work with reminders"
+        "the way reminders in {APP_NAME_COMPOSE} look and behave which provides an easier way for the user to work with."
```

Comment:
Is ok for now, but let's try to be less vague when describing things -- we can probably point out a few key changes as an example.
We should also consider expanding the plugin to support images/videos (example would be [vscode](https://code.visualstudio.com/updates/v1_79))